runtime/browser/xwalk_browser_main_parts_android.cc

```diff
@@ -188,6 +189,8 @@ void XWalkBrowserMainPartsAndroid::PreMainMessageLoopRun() {
 
   extension_service_ = xwalk_runner_->extension_service();
 
+  content::RenderFrameHost::AllowInjectingJavaScriptForAndroidWebView();
```

Comment:
Just one request before merging: can you please add a comment here mentioning crbug.com/507809 and saying we need to call this because we provide `evaluateJavaScript` like the Android WebView?